### DISCUSSION TITLE
Re-interant routing

### DISCUSSION THREAD

**Discussion by kareefardi** [The-OpenROAD-Project/OpenROAD#5888] [Q&A]
Given that OR can do antenna repair post-DRT, does that mean that routing is now re-interant. For example, after doing some ECOs, would OR be able to only route specific nets. If that's the case, is there any where else in OR where this can be utilized and different usecases - perhaps better timing closure for example?

**Comment by maliberty** [comment]
You rarely want to disturb the layout after detailed routing.  We mostly plan to look at Vt or other identical footprint swaps.
